app/build.gradle

```diff
@@ -29,4 +33,7 @@ dependencies {
     implementation 'com.madgag.spongycastle:prov:1.54.0.0'
     implementation 'edu.ucar:jj2000:5.2'
     implementation 'com.github.mhshams:jnbis:1.1.0'
+    implementation 'org.apache.commons:commons-text:1.8'
```

Comment:
Let's remove Apache commons dependency for now. If you want you can split it into a separate PR.